chromatic/rainbows/visualizations/plot_noise_comparison.py

```diff
@@ -0,0 +1,128 @@
+from ...imports import *
+
+__all__ = ["plot_noise_comparison"]
+
+
+def plot_noise_comparison(
```

Comment:
This is great! I changed a few variable names and a couple of the defaults, but this is pretty perfect. Thank you!